Release of API version 1.0.0-rc.1 (#121)
camaraproject/NumberVerification: documentation/API_documentation/NumberVerification-Readiness-Checklist.md

# API Readiness Checklist

> Checklist for number_verification in version b1.0.0-rc1.1

Reviewer: ```suggestion
Checklist for number_verification 1.0.0-rc.1 in release r1.1
```

| Nr | API release assets  | alpha | release-candidate |  public-release<br>initial | public-release<br> stable | Status | Comments |
|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|

> |  1 | API definition                               |   M   |         M         |    M    |    M   |   Y   | [link](https://github.com/camaraproject/NumberVerification/blob/main/code/API_definitions/number_verification.yaml) |

Reviewer: Best practice is to use relative URL (which are pointing then to the current branch or tag):

```suggestion
|  1 | API definition                               |   M   |         M         |    M    |    M   |   Y   | [link](/code/API_definitions/number_verification.yaml) |

```

|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |  Y    |      |
|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |   Y   |      |
|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |   Y   |      |

> |  5 | API documentation                            |   M   |         M         |    M    |    M   |   Y   | Embed documentation into API spec - [link](https://github.com/camaraproject/NumberVerification/blob/main/code/API_definitions/number_verification.yaml)  |

Reviewer: ```suggestion
|  5 | API documentation                            |   M   |         M         |    M    |    M   |   Y   | Embed documentation into API spec - [link](/code/API_definitions/number_verification.yaml)  |

```

|  6 | User stories                                 |   O   |         O         |    O    |    M   |   N   | [PR118](https://github.com/camaraproject/NumberVerification/pull/118) |
|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |   N   | link |
|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |   N   | link |
|  9 | Test result statement                        |   O   |         O         |    O    |    M   |   N   | link |
| 10 | API release numbering convention applied     |   M   |         M         |    M    |    M   |   Y   |      |

> | 11 | Change log updated                           |   M   |         M         |    M    |    M   |   N   | [PR121](https://github.com/camaraproject/NumberVerification/pull/121) |

Reviewer: It this PR get merged, the link will be:
```suggestion
| 11 | Change log updated                           |   M   |         M         |    M    |    M   |   N   | [link](/CHANGELOG.md) |

```
